### ISSUE TITLE
Publish operation hangs when some connection step details are opened

### ISSUE THREAD

**Issue by christophd** [syndesisio/syndesis#4792]
## This is a...

<pre><code>
[ ] Feature request
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Documentation issue or request
</code></pre>

## Description

- Open an integration
- select a connection to display its details
- While on that opened details page hit the publish button 

The publish operation hangs and nothing happens. This only happens when you hit that publish button while a connection step details page is opened.

![out](https://user-images.githubusercontent.com/195264/53967008-b090fa80-40f4-11e9-9c6e-b07d25c14b8e.gif)

**Comment by gashcrumb** [comment]
I'll take a look!

**Comment by gashcrumb** [comment]
BTW, any info in the javascript console?

I tried this here and I cannot reproduce the behavior, when Publish is clicked I can see a few info messages on the console + requests going out and I'm back at the integration list view.  I had rebuilt this morning to test it out against a build of the UI imageand also tried reproducing using the development server too.

Maybe give it a test from an incognito or open up the dev console and ensure in it's settings that 'Disable cache when DevTools are open' is turned on and see if that changes anything?
